purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1051,7 +1051,7 @@ class Purchases internal constructor(
             for (j in 0 until jsonPackagesArray.length()) {
                 skus.add(
                     jsonPackagesArray.getJSONObject(j)
-                        .getString("platform_product_identifier")
+                        .getString("platform_product_group_identifier")
```

Comment:
this might have to be optString as we are migrating on the backend

Reply:
+1, if it is'n present we should just ignore it and log an error (although this shouldn't ever happen)

Reply:
good catch